src/test/java/com/crud/tasks/mapper/TrelloMapperTest.java

```diff
@@ -0,0 +1,104 @@
+package com.crud.tasks.mapper;
+
+import com.crud.tasks.domain.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class TrelloMapperTest {
+
+    @Autowired
+    private TrelloMapper trelloMapper;
```

Comment:
jw. jesli mozna uzyc 'czystego junita' lub z mockito to lepiej unikac kontenera spring'owego. Testy beda szybsze